tests/socket.rs

```diff
@@ -1607,7 +1610,27 @@ fn original_dst() {
 }
 
 #[test]
-#[cfg(all(feature = "all", any(target_os = "android", target_os = "linux")))]
+#[cfg(all(feature = "all", target_os = "windows"))]
```

Comment:
The tests should be the same across OS, so please merge it with the `original_dst` above. Same for `original_dst_ipv6`.